.devcontainer/post-start.sh

```diff
@@ -1,5 +1,14 @@
 #!/bin/sh
 
+echo "Ensuring ruby is up to date..."
+export PATH="${HOME}/.asdf/shims:${HOME}/.asdf/bin:${PATH}"
+asdf plugin update ruby
+asdf install ruby $( cat .ruby-version )
+asdf global ruby $( cat .ruby-version )
```

Comment:
This is redundant, no? This is already in on-create.sh.

Reply:
Usually redundant, but not when `.ruby-version` has been updated. (Which I ran into the other day) It's very quick if the version is up to date, but I could also wrap it in a `if (installed_version !== vets_api_ruby_version)` check...

Or we can say "you should just create a new codespace", but this feels like a nice way to improve the developer experience...